samples/EventImporter/Consumer.Tests/pacts/Event API Consumer V3 Message-Event API V3 Message.json

```diff
@@ -6,15 +6,24 @@
     {
       "contents": [
         {
-          "EventId": "45d80d13-d5a2-48d7-8353-cbb4c0eaabf5",
-          "EventType": "SearchView",
-          "Timestamp": "2014-06-30T01:37:41.0660548"
+          "eventId": "45d80d13-d5a2-48d7-8353-cbb4c0eaabf5",
+          "eventType": "SearchView",
+          "timestamp": "2014-06-30T01:37:41.0660548"
         }
```

Comment:
Hmmm something has gone wrong here because the casing has changed. I'm not sure whether it was wrong previously or now though (on my phone at the moment)

Reply:
Yep, see #351